### PR TITLE
[SE-4304] Cherry-pick add celerybeat configuration

### PR DESCRIPTION
**Pull request by gabor-boros**
This PR cherry-picks commits from PR https://github.com/edx/configuration/pull/6407 and https://github.com/edx/configuration/pull/6442.

The first PR added was responsible for adding a custom scheduler configuration while the second PR fixes that in a way to let workers scale.

**Dependencies**: None

**Merge deadline**: ASAP

**Testing instructions**:

* Proofread the configuration changes

**Author notes and concerns**:

_The PRs were already tested._

**Reviewers**
- [ ] @jdmulloy -- sorry for directly mentioning you on this PR, but you have the most context as you approved both previous PRs. If you'd like me to find another reviewer, please let me know!

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @gabor-boros! I've created [OSPR-5872](https://openedx.atlassian.net/browse/OSPR-5872) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@gabor-boros Thank you for your contribution. This is ready for our review, correct?

**Comment by gabor-boros** [comment]
@natabene I'll create an instance based on this configuration to test instance creation, but yes, this should be ready for review. 

Side note: A reviewer is already mentioned in the description who has the most context.

**Comment by jdmulloy** [comment]
@gabor-boros Can you please add the last two commits that are on #6455 to this one so that we're merging the same changes for koa and lilac?

Specifically these commits 
https://github.com/edx/configuration/pull/6455/commits/1975b4ca7dd31312847e2ca2b5f547b50eb719d6
https://github.com/edx/configuration/pull/6455/commits/6caefce3714a9a8cd0aeaba0f7e9311e4ffad15b

**Review by jdmulloy — CHANGES_REQUESTED** [review]
@gabor-boros Can you please add the last two commits that are on #6455 to this one so that we're merging the same changes for koa and lilac?

Specifically these commits
1975b4c
6caefce

**Comment by jdmulloy** [comment]
@gabor-boros Please also make sure that the fix in this PR makes it onto this branch before we merge. https://github.com/edx/configuration/pull/6467/files

**Comment by gabor-boros** [comment]
@jdmulloy This is ready for your review. Thanks for the heads up about the other PR!

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 1]
```diff
@@ -357,13 +357,32 @@
...
+    version: "{{ EDXAPP_SINGLE_BEAT_VERSION|default(omit) }}"
+    virtualenv: "{{ edxapp_venv_dir }}"
+    state: present
+  become_user: "{{ edxapp_user }}"
+  when: EDXAPP_ENABLE_CELERY_BEAT
+  tags:
+    - install
+    - install:app-requirements
+
+- name: import custom tinymce plugins
```
Comment (jdmulloy) [inline playbooks/roles/edxapp/tasks/deploy.yml]: Is this related to the celery beat stuff, or is it something form master that accidentally made it over to this cherry pick branch?

This is the PR that added tinyMCE #6179

**Review by jdmulloy — CHANGES_REQUESTED** [review]

File: playbooks/roles/edxapp/tasks/deploy.yml (hunk 1 above)
Comment (gabor-boros) [reply playbooks/roles/edxapp/tasks/deploy.yml]: @jdmulloy  Sorry, that was something from master that accidentally sneaked in. Removed that.

**Review by jdmulloy — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@gabor-boros 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/6456&entry.1484655318=https://github.com/gabor-boros&entry.752974735=2021-06-21+08:10&entry.1917517419=2021-07-15+15:11&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**